src/fem/mapping_p1.cc

```diff
@@ -298,7 +298,17 @@ auto MappingP1<dim,spacedim>::clip_to_element(BaryPoint &barycentric) -> BaryPoi
     return RefElement<dim>::clip(barycentric);
 }
 
-
+template<unsigned int dim, unsigned int spacedim>
+bool MappingP1<dim,spacedim>::is_point_inside(const BaryPoint &bp){
```

Comment:
There is same functionality in Element, we should deduplicate this. The right place is in Mapping.

Reply:
solved